proseco/tests/test_fid.py

```diff
@@ -10,7 +10,9 @@
 from ..fid import get_fid_catalog, get_fid_positions
 from .test_common import DARK40, OBS_INFO, STD_INFO, mod_std_info
 
-# Reference fid positions for spoiling tests
+# Reference fid positions for spoiling tests.  Because this is done outside
+# a test the fixture to generally disable fid drift is not in effect.
+os.environ["PROSECO_ENABLE_FID_OFFSET"] = "False"
```

Comment:
Use the [temp_env_var](https://sot.github.io/ska_helpers/#ska_helpers.utils.temp_env_var) context manager here.

I'm not happy about the `agasc.SUPPLEMENT_ENABLED_ENV` line below either, but that is for another day.

Reply:
I wasn't sure how much cleanup to do while I was in here, but it seems like both of these should just move into fixtures.